### DISCUSSION TITLE
Repository settings branches tab is missing controls to switch the default branch

### DISCUSSION THREAD

**Discussion by Mattia-Re** [community/community#101698] [Repositories] [answered]
### Select Topic Area

Question

### Body

I want to switch the default branch of my repository. My repo has 2 branches and when i navigate to `https://github.com/iKingNinja/repo/settings/branches` this is all I see.

![immagine](https://github.com/community/community/assets/83887130/6c5ddcb0-47f3-4979-9c8e-9b09170f9644)

**Comment by seyLu** [comment, marked as answer]
#### Hi @iKingNinja,

Changing the default branch is located on Settings > General.
![Screenshot from 2024-01-24 04-18-55](https://github.com/community/community/assets/98249191/b328613b-e49d-4d08-a0cf-2be12a91e84e)
<br>

You can read more on the [GitHub docs](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch#changing-the-default-branch).

**Reply by Mattia-Re** [reply]
Oh right, for some reason I thought the docs said it was under the branches tab.
